catalog/solr/pom.xml

```diff
@@ -30,6 +30,11 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.7</version>
             </dependency>
+            <dependency>
+                <groupId>commons-fileupload</groupId>
+                <artifactId>commons-fileupload</artifactId>
+                <version>1.3.2</version>
```

Comment:
Believe you should use ${commons.fileupload.version} here which can be inherited from the catalog/pom.xml. Any reason not to do that?